### ISSUE TITLE
[Policy Based Recording] One bot per the set of users who have the same policy

### ISSUE THREAD

**Issue by pavanjains** [microsoftgraph/microsoft-graph-comms-samples#502]
We have a scenario where if say 3 different people, who are all covered by a policy in some way to require compliance recording, and they join the same meeting, bot is called 3 separate times.   
We had originally thought we’d be asked to record once (since it is one meeting), but it is more that there are 3 individuals of interest, and so each trigger a request for the bot to join and record, resulting in potentially recordings that contain the same information… is that correct? Or is it that we’re not crafting our policy/policies correctly?

Is there any bot grouping that can be done to optimize it as one BOT per the set of user who have the same policy?

**Comment by 1fabi0** [comment]
yes it is correct because, for me at least, it seems like the easiest solution for microsft to implement the policies. Also you can then easily save recordings according to the user who should be recorded.

**Comment by vidilab** [comment]
We see the same. The scenario id of the call seems usable to detect that the calls are related and manually implement not to start.
